Resources/Prototypes/Entities/Structures/Walls/walls.yml

```diff
@@ -712,4 +712,32 @@
       - FullTileMask
       layer:
       - GlassLayer
-  - type: Airtight
\ No newline at end of file
+  - type: Airtight
+
+- type: entity
+  id: WallForce
+  name: Force Wall
+  components:
+    - type: TimedDespawn
+      lifetime: 20
+    - type: Tag
+      tags:
```

Comment:
Bit suss it doesn't inherit from WallBase to me (yes the mime wall is also very suss) as it's missing some stuff, i.e. anchoring, sprite netsync false, and I don't think that icons needed.